src/libs/ReportUtils.ts

```diff
@@ -3596,8 +3596,8 @@ function buildOptimisticIOUReport(payeeAccountID: number, payerAccountID: number
     const payerEmail = 'login' in personalDetails ? personalDetails.login : '';
 
     const participants: Participants = {
-        [payeeAccountID]: {hidden: false},
-        [payerAccountID]: {hidden: false},
+        [payeeAccountID]: {hidden: true},
+        [payerAccountID]: {hidden: true},
```

Comment:
I initially thought here we should also set hidden value based on some flag as `buildOptimisticIOUReport` called from multiple flows but seems not needed and this works fine.